Feature ministries

flaaviu79 opened the pull request:
Added logo images
Added buttons
Added buttons Titles
Added buttons Subtitles
Added buttons Setas

flaaviu79 committed:
Added logo images to assets
Added buttons on ministries screen
Added button title
Added button subtitle
Added button seta

flaaviu79 committed:
Added Logo Image to assets

saabrinakim commented:
Ótimo trabalho, @flaaviu79! Alguns detalhes apenas. Os componentes deveriam começar la de cima da página e não no meio. E a segunda linha de descrição do ministério está um pouco mais pra esquerda do que a primeira linha.

flaaviu79 committed:
Modified marginright after logo image
Deleted space at the start and end of every text
Modified content, textContainer and buttonSeta style
